### ISSUE TITLE
Error: No NEX password set for PID

### ISSUE THREAD

**Issue by EmilianoSandri** [PretendoNetwork/nex-viewer#25]
I'm trying to use nex-viewer to parse some packet captures I've done on Gen 6 Pokèmon games but after successfully parsing few packets the simple script I'm trying to use (basically copied from repo README.md) keeps failing with the errors you see in the 2nd screenshot, the strangest thing is that I have a single nex account in my nex-keys.txt and its username is not 2000021103, in fact this string doesn't appear anywhere in my $HOME

![Screenshot from 2024-03-12 00-53-52](https://github.com/PretendoNetwork/nex-viewer/assets/128513458/53610447-9fb1-4d30-a212-e6273473e18c)
![Screenshot from 2024-03-12 00-54-37](https://github.com/PretendoNetwork/nex-viewer/assets/128513458/dc31c706-fbf0-4dfe-ae79-067263bde289)

Can this be caused by the fact that the save on which I'm experimenting was started on another console and then transferred on the other console trough Checkpoint?

**Comment by jonbarrow** [comment]
> Can this be caused by the fact that the save on which I'm experimenting was started on another console and then transferred on the other console trough Checkpoint?

No. Your PID is stored in a sysmodule save. You just haven't dumped the right PID/password it seems

**Comment by jonbarrow** [comment]
Closing because this is now 2 years stale and the entire app has been rewritten since then
